src/client/extension.ts

```diff
@@ -162,6 +162,13 @@ export async function activate(
             (fl) => ({ ...fl, uri: fl.uri.fsPath } as WorkspaceFolder)
         ) || [];
     
+    const data = getPortalsOrgURLs(workspaceFolders);
```

Comment:
Will this also handle any update to workspace?

Reply:
On every change in our workspace folder, this code path is getting triggered